vnclauncher.py

```diff
@@ -106,14 +106,18 @@ def __init__(self, handle):
 
         self._vte = VTE()
         self._vte.show()
-
+        self._vte.connect("child-exited", self._quit_cb)
         box.pack_start(self._vte, True, True, 0)
         box.pack_start(table, False, False, 0)
 
         self.set_canvas(box)
         box.show()
         self._vte.grab_focus()
 
+    def _quit_cb(self, vte, data=None):
+        print("Quitting...")
```

Comment:
Why print? when logging calls were used in the activity.

Reply:
Removed.